packages/palette-charts/src/elements/BarChart/Bar.tsx

```diff
@@ -12,14 +20,19 @@ const BAR_HEIGHT_RANGE = MAX_BAR_HEIGHT - MIN_BAR_HEIGHT
 
 interface BarBoxProps {
   isHighlighted?: boolean
+  primaryColor?: Color
+  hoverColor?: Color
+  highlightColor?: Color
 }
 
 // the actual visible bit of the bar
 const BarBox = styled(Box)<BarBoxProps>`
   transition: height 0.8s ease;
   position: relative;
-  background: ${(props: BarBoxProps) =>
-    props.isHighlighted ? color("mono60") : color("mono10")};
+  background: ${(props) => 
+    props.isHighlighted 
+      ? color(props.highlightColor || "mono60")
+      : color(props.primaryColor || "mono10")};
```

Comment:
`color` is [deprecated](https://github.com/artsy/palette/blob/main/packages/palette/src/helpers/color.ts#L7) (and probably should just be deleted at this point). This won't work with dark mode. Use `themeGet` instead.

Reply:
to confirm: themeGet takes string as param, so I should update primaryColor to be a string instead of Color, right?